### PR TITLE
feat: Add line-numbers keyword

### PR DESCRIPTION
**Pull request by fxwiegand**
This PR adds the option to include line numbers in a datavzrd report by adding the new line-numbers keyword to a view defintion.

### CODE REVIEW

File: web/src/datavzrd.js [hunk 1]
```diff
@@ -68,6 +68,7 @@ function renderTickPlot(ah, columns, title, slug_title, specs, is_float, precisi
     if (detail_mode || header_label_length !== 0) {
         index += 1;
     }
+    index += 1; // compensate for line numbers
```
Comment (johanneskoester) [inline web/src/datavzrd.js:71]: This looks repetetive and easy to forget

**Review by johanneskoester — CHANGES_REQUESTED** [review]

File: web/src/datavzrd.js (hunk 1 above)
Comment (fxwiegand) [reply web/src/datavzrd.js]: I changed the index when initializing it. Looks cleaner now thanks!

File: web/src/datavzrd.js [hunk 2]
```diff
@@ -146,7 +146,7 @@ function renderDetailTickBarPlot(value, div, specs, title) {
 }
 
 function colorizeColumn(ah, columns, heatmap, detail_mode, header_label_length) {
-    let index = columns.indexOf(heatmap.title) + 1;
+    let index = columns.indexOf(heatmap.title) + 2;
```
Comment (johanneskoester) [inline web/src/datavzrd.js:149]: I'd rather have a function `getIndex(name: str) -> int` which does this.

Comment (fxwiegand) [reply web/src/datavzrd.js]: Done!

**Pull request merged**